src/components/PhoneInput/index.tsx

```diff
@@ -80,10 +79,6 @@ const StyledLabel = styled.label<PhoneInputLabelProps>`
     .iti__selected-dial-code {
       padding-left: 5px;
     }
-
-    .iti__flag {
-      background-image: url(${flags});
```

Comment:
The issue is that it won't work on project that use scw-ui as flags provided by intl-tel-input are relative paths, we need to explicitly tell it where to find the flags

https://github.com/jackocnr/intl-tel-input#getting-started-using-a-bundler-eg-webpack

Reply:
Fixed by using the `flags.png` image instead of `flags@2x` that was used as `flags.png`

Reply:
So we loose support for high dpi devices ? 😄

Reply:
Pushed a commit that adds back the `flags@2x` image, so we're back to the state before.